src/query/functions/temporal/rate.go

```diff
@@ -139,20 +143,18 @@ func standardRateFunc(
 		counterCorrection   float64
 		firstVal, lastValue float64
 		firstIdx, lastIdx   int
-		firstTS, lastTS     time.Time
+		firstTS, lastTS     int64
```

Comment:
Can we use `xtime.UnixNano` here too? https://github.com/m3db/m3/blob/master/src/x/time/unix_nano.go#L27